receptorctl/sonar-project.properties

```
@@ -0,0 +1,48 @@
+# SonarQube Cloud project configuration for Receptorctl
+# Complete documentation: https://docs.sonarqube.org/latest/analysis/analysis-parameters/
+
+# =============================================================================
+# PROJECT IDENTIFICATION (REQUIRED)
+# =============================================================================
+
+# The unique project identifier. This is mandatory.
+# Do not duplicate or reuse!
+# Available characters: [a-zA-Z0-9_:\.\-]
+# Must have least one non-digit.
+sonar.projectKey=ansible-automation-platform_receptorctl
+sonar.organization=ansible
+
+# Project metadata
+sonar.projectName=receptorctl
+
+# =============================================================================
+# SOURCE AND TEST CONFIGURATION
+# =============================================================================
+
+# Source directories to analyze
+sonar.sources=.
+sonar.exclusions=\
+    **/tests/**
+
+# Test directories
+sonar.tests=tests
+
+# Set branch-specific new code definition
+#
+# This is important to always check against the main branch for new PRs,
+# otherwise the PR may fail during backporting, since the old version of the code
+# may not respect the minimum requirements for the existing Quality Gate.
+sonar.newCode.referenceBranch=devel
+
+# =============================================================================
+# LANGUAGE CONFIGURATION
+# =============================================================================
+
+# File encoding
+sonar.sourceEncoding=UTF-8
+
+# =============================================================================
+# REPORTS AND COVERAGE
+# =============================================================================
+
+sonar.python.coverage.reportPaths=receptorctl_coverage.xml
```

```suggestion
sonar.python.coverage.reportPaths=receptorctl_coverage.xml

```
Adding a new line here as well.